src/callbacks/onDraw/sortYdomain.js

```diff
@@ -1,29 +1,25 @@
-import { nest, min } from 'd3';
+import { nest, min, set } from 'd3';
 
 export default function sortYdomain() {
-    //Sort y-domain by the earliest event of each ID.
-    if (this.config.y.sort === 'earliest') {
-        //Redefine filtered data as it defaults to the final mark drawn, which might be filtered in
-        //addition to the current filter selections.
-        const filtered_data = this.raw_data.filter(d => {
-            let filtered = false;
-
-            this.filters.forEach(di => {
-                if (
-                    filtered === false &&
-                    di.val !== 'All' &&
-                    d[this.config.event_col] !== 'Grouping'
-                ) {
-                    filtered =
-                        di.val instanceof Array
-                            ? di.val.indexOf(d[di.col]) === -1
-                            : di.val !== d[di.col];
-                }
-            });
+    //Redefine filtered data as it defaults to the final mark drawn, which might be filtered in
```

Comment:
this blew my mind a bit, but going to take your word for it since sort is working well 😸

Reply:
As I recall if two marks have different `values` objects then the chart's `filtered_data` array is filtered on the second mark's `values` specification, in addition to any filtering from controls.